### ISSUE TITLE
Feature Request: Plaatjes in nieuws artikelen

### ISSUE THREAD

**Issue by Jelle-DIVD** [DIVD-NL/web-www-v2#629]
I have been working with Serena on some new articles and we found that embedding images is problematic.

I will try to gather some more info on how, where and what, but it seems not even embedding a data base64 encoded image would work.

We'd obviously need some form of storage/upload endpoint. I would love it it it can be just 'paste in-line' like here in a github issue.

**Comment by martendegroot** [comment]
Good point there! I know we're already uploading some media.... but honestly wouldn't have a clue how. Perhaps @MagicLegend  can enlighten us here. Just to be clear, @Jelle-DIVD , you want to place this in the content right? That's all in markdown, so a self-hosted image should work as well I guess, although that sounds far from ideal (and is that actually very safe?)

**Comment by Jelle-DIVD** [comment]
Correct. Ideally we should have an upload endpoint for editors that their uploads will POST to, which validates, resizes and does all that jazz, and it'll just return an image url after the post so it can be embedded in markdown

**Comment by MagicLegend** [comment]
![image](https://github.com/user-attachments/assets/bdb1ff4d-691a-45c9-ac47-9a90491e347c)

Media management lives here in the CMS. Then the image can be used using standard Markdown to include the image wherever it's desired: `![alt text](Isolated.png "Title")`. Note that if the image is not used in the Markdown, it gets optimized away and is not hosted.

**Comment by imtiaz-divd** [comment]
Does this task still need any attention @Marieke-DIVD?

**Comment by Marieke-DIVD** [comment]
Yes it does! 

Is it possible to follow the design in Figma on this matter? So if we add an image that it automagically has rounded corners with radius of 25?

<img width="559" alt="Image" src="https://github.com/user-attachments/assets/e1d59605-d9c7-4059-9277-67769d10aca5" />

**Comment by Marieke-DIVD** [comment]
Adding docs:

![Image](https://github.com/user-attachments/assets/3afa98ef-426b-429b-88de-672a06835964)
